src/backend/db/products.js

```diff
@@ -8,23 +8,173 @@ import { v4 as uuid } from "uuid";
 export const products = [
   {
     _id: uuid(),
-    title: "You Can WIN",
-    author: "Shiv Khera",
-    price: "5000",
-    categoryName: "non-fiction",
+    name: "balsam",
+    categoryName: "perennial",
+    image: "/perennial/balsam.webp",
+    price: "100",
+    discount: "10",
```

Comment:
suggestion: price shouldn't be in double quote because you have to do mathematical operation on it

Reply:
Thanks for the suggestion, I will surely fix this